src/intents.cs

```diff
@@ -7595,10 +7749,20 @@ interface INWorkoutLocationTypeResolutionResult {
 	[BaseType (typeof (NSUserActivity))]
 	interface NSUserActivity_IntentsAdditions {
 
+		[Mac (12,0)]
```

Comment:
This is strange: this method already existed, but Apple marked it as available only in macOS 12.0 now?

Reply:
yep - this is the case for most of the updates in this PR. 

```diff
-@property (readonly, nullable, NS_NONATOMIC_IOSONLY) INInteraction *interaction API_AVAILABLE(ios(10.0), watchos(3.2), tvos(14.0)) API_UNAVAILABLE(macos);
+@property (readonly, nullable, NS_NONATOMIC_IOSONLY) INInteraction *interaction API_AVAILABLE(ios(10.0), macos(12.0), watchos(3.2), tvos(14.0));
```

Reply:
Ah, I see, it was marked as unavailable in Apple's headers, but we never reflected that in our bindings (so our attributes were incorrect, and now they aren't anymore).